src/node/transaction.cpp

```diff
@@ -55,6 +55,9 @@ TransactionError BroadcastTransaction(const CTransactionRef tx, std::string& err
 
         // Transaction was accepted to the mempool.
 
+        // the mempool explicitly keeps track of wallet txns to ensure succesful initial broadcast
```

Comment:
nit: this is also for transactions sent by `sendrawtransaction`

Reply:
hm, could I say "my" transactions, does that cover the local cases? or should I just be more explicit and say "submitted via wallet or sendrawtransaction"

Reply:
I'd suggest "submitted locally by the wallet or RPC"